src/api_keys/api.rs

```diff
@@ -78,16 +79,27 @@ pub async fn make_request(
     let url = user_make_request_info
         .api_url
         .replace("xxxREPLACE_MExxx", &api_key_info);
+
+    let mut headers = HeaderMap::new();
+    for (key, value) in &user_make_request_info.http_headers {
+        let first = key.replace("xxxREPLACE_MExxx", &api_key_info);
```

Comment:
No harm in leaving this in, but it would be unconventional to put the API key in the header name

Reply:
ya same, just cuz I can't think of the answer doesn't mean it doesn't exist, so I was like no harm of having it